nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/migration/MigrationConstants.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour AL KOTOB
+ */
+package org.nuxeo.rest.management.migration;
+
+/**
+ * @since 11.2
```

Comment:
```suggestion
 * @since 1.0.1
```

Same for all `@since` in this PR :)

Reply:
🤦